server/services/nl.py

```diff
@@ -57,7 +259,88 @@ def _download_embeddings(self):
       _, filename = os.path.split(blob.name)
       blob.download_to_filename(os.path.join(TEMP_DIR, filename))  # Download
 
-  def search(self, query):
+  def _get_related_places(self, place_dcid):
+    place_page_data = dc.get_landing_page_data(place_dcid, 'Overview', [])
+    if isinstance(place_page_data, dict):
+     return place_page_data
+    return {}
+
+  def _get_svg_info(self, entities, svg_dcids):
+    result = dc.get_variable_group_info_bulk(svg_dcids, entities)
+    if isinstance(result, dict):
+      return result
+    return {}
+
+  def _related_svgs(self, svs_list, relevant_places):
+    # Using PropertyValues (memberOf) and VariableGroupInfo APIs
+    svs_to_svgs = dc.property_values(svs_list, 'memberOf')
+
+    # Get all distinct SVGs and SV under verticals.
+    svgs = set()
+    sv_under_verticals = set()
+
+    for sv, svg_list in svs_to_svgs.items():
+      has_svg = False
+      for svg in svg_list:
+        svgs.add(svg)
+        if '_' in svg:
+          has_svg = True
+      if not has_svg:
+        # This is top-level SV, so we will try getting child SVGs
+        sv_under_verticals.add(sv)
+    
+    # Get SVG info for all relevant places
+    svgs_info = self._get_svg_info(relevant_places, list(svgs))['data']
+    return svgs_info
+
+
+  def _related_places(self, dcid):
+    # Get related places using Place API
+    related_places = self._get_related_places(dcid)
+    related_places['nearbyPlaces'] += [dcid]
+    related_places['similarPlaces'] += [dcid]
+
+    return related_places
+
+  def _peer_buckets(self, sv2definition, svs_list, fixed_prefixes):
```

Comment:
nit: feel free to ignore, but the fixed_prefixes here and fixed_props in _chart_config could perhaps be removed in place of using FIXED_PROPS, FIXED_PREFIXES global (since they are constants)

Reply:
Using the global consts everywhere now